_imagingft.c

```diff
@@ -93,6 +122,78 @@ geterror(int code)
     return NULL;
 }
 
+static int
+setraqm(void)
+{
+    /* set the static function pointers for dynamic raqm linking */
+    p_raqm.raqm = NULL;
+
+    /* Microsoft needs a totally different system */
+#if !defined(_MSC_VER)
```

Comment:
Instead of: 
#if !something 
a 
#else
b
#endif

How about this? 
#if something 
b 
#else 
a 
#endif

Reply:
Windows is the weird one. Everyone else follows the posix path, even people on msys2.

(and frankly, unless I can actually test with windows, I'm potentially going to drop the second clause. I was ok with shipping code if it was compiled and staticly linked, but doing it at runtime has a much larger chance of doing something bad)